portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,12 @@ function addRandomGreeting() {
 function changeBackgroundColor(color) {
     document.body.style.background = color;
 }
+
+/**
+ * Fetches 'Hello Betty!' message from the server and adds it to the DOM.
+ */
+async function getHelloMessage() {
```

Comment:
Let's make this function name a bit more generic, same for the the comment. This helps us prevent having to change the name of the function / comment on every change.

I'm thinking: `getGreetingMessage()` and a more generic comment above.